core/context/src/main/java/com/milesight/beaveriot/context/security/SecurityContextUser.java

```diff
@@ -0,0 +1,21 @@
+package com.milesight.beaveriot.context.security;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author loong
+ * @date 2024/12/5 13:00
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+@Documented
+@Inherited
+public @interface SecurityContextUser {
```

Comment:
Whether to consider adding additional SecurityUser attributes, such as the user ID

Reply:
The user ID exists in the context and does not require additional construction settings

Reply:
In the scenario where the user ID has already been logged in, it will exist in the context without the need for additional construction. In non logged in situations (such as registration), the user ID has not yet been generated. So currently, there is no need for the usage scenario of userId